src/style/shadows.scss

```diff
@@ -109,3 +109,105 @@
         -1px -1px 0 1px rgba(var(--color-black), 0.07) inset,
         0 0 1px 1px rgba(var(--color-white), 0.98) inset;
 }
+
+@media (prefers-color-scheme: dark) {
```

Comment:
Due to what I explained in an earlier comment, I have not used any magic like 
````scss
:root[data-theme="force-dark"]` {
````
or 
````scss
@media (prefers-color-scheme: dark) {
    :root:not([data-theme="force-light"]) {
````

Not sure if that's the right way forward. 
If it is, then we need to use some `mixin` instead of copy/pasting the code into each block.

Reply:
I think we need a mixin then, because I really think we'll want to have support for a button to switch between light and dark mode… For one thing, I really want support for it in the docs, because I don't want to have to switch my system setting every time I want to test a component, but secondly, I think we'll want to have some setting like that in the Lime CRM web client too (but as I said, even if we wouldn't, I still want it in the docs).

Reply:
We absolutely has to support the toggle button. But my question was, is this the only way to override `prefers-color-scheme: dark`? Can't we fake it and say to the browser that:
> No no... right now the user doesn't prefer dark color-scheme. Please ignore whatever they OS says, and run the light color-scheme.

If there was a way for this, we wouldn't have to make a duplicate copy of everything then.

I haven't googled it yet. But it's worth to give it a try.

Reply:
I don't think that's possible. I think I googled that a bit when we added dark mode support to Kompendium… but it's absolutely worth looking into 👍

By the way, we have a related issue in the _Color system_  design guideline document. The highlighting of the brand colours determines which swatches to highlight based only on the system setting for dark mode, and doesn't account for the user's setting:

https://user-images.githubusercontent.com/154725/210375826-12396b8e-d0bd-4064-8b4f-4dcd32c3e2d2.mov

Reply:
Ok. cool. I think until we figure out whether there is another way or not, I'll copy the mixin that you created for Kompendium and use it here in Elements.

We will look into the issue you mention as well.

Reply:
### Other known issues

- lock icon is hard to see 
    <img width="684" alt="image" src="https://user-images.githubusercontent.com/35954987/211520527-fff75ecd-671e-4df6-b0f7-fee9137140bf.png">